addons/mail/static/src/models/composer/composer.js

```diff
@@ -567,6 +537,13 @@ function factory(dependencies) {
         isDoFocus: attr({
             default: false,
         }),
+        /**
+         * If true composer will log a note, else a message (in a functional way of saying)will be
```

Comment:
If we keep it like this: space after )

But is the doc supposed to be technical or functional here @alexkuhn ? Maybe we can just call it a "comment" instead of message.

Reply:
Indeed "comment" seems more appropriate. Even if UI and placeholder uses term "message", it's a bit confusing to use this as a "mode" for posting a message. We plan to model in-progress message from composer (task-2283218), so the term "comment" will make much more sense by being closer to resulting `subtype` of message.